src/overlays/actors/ovl_En_Reeba/z_en_reeba.h

```diff
@@ -15,17 +15,17 @@ typedef struct EnReeba {
     /* 0x01FC */ Vec3s morphTable[18];
     /* 0x0268 */ char unk_268[0x4];
     /* 0x026C */ EnReebaActionFunc actionfunc;
-    /* 0x0270 */ s16 unk_270;
-    /* 0x0272 */ s16 unk_272;
-    /* 0x0274 */ s16 unk_274;
-    /* 0x0276 */ s16 unk_276;
-    /* 0x0278 */ s16 unk_278;
+    /* 0x0270 */ s16 bigLeeverTimer; // big Leever uses to close and make distance from Link.
+    /* 0x0272 */ s16 surfaceTimer;
+    /* 0x0274 */ s16 sfxTimer; // delay between move sounds. 2x if big Leever.
+    /* 0x0276 */ s16 painTimer;
+    /* 0x0278 */ s16 waitTimer;
     /* 0x027A */ s16 isBig;
-    /* 0x027C */ s16 unk_27C;
-    /* 0x027E */ s16 unk_27E;
-    /* 0x0280 */ s16 unk_280;
-    /* 0x0284 */ f32 unk_284;
-    /* 0x0288 */ f32 unk_288;
+    /* 0x027C */ s16 unkDamageField; // set when hit with ice and most other weapons. Never read.
+    /* 0x027E */ s16 stunType;
+    /* 0x0280 */ s16 aimType; // inc'd by spawner. dictates rotateY modification
```

Comment:
```suggestion
    /* 0x0280 */ s16 aimType; // incremented by the spawner. Dictates rotateY modification
```
I wasn't sure what this abbreviation was, had to look up the code. I think we can afford "incremented" here.